src/components/EditTask.js

```diff
@@ -29,7 +32,7 @@ class EditTask extends PureComponent {
       timestamp,
       comment,
       priority,
-    } = this.state
+    } = this.state.task
 
     const { edit, history } = this.props
 
```

Comment:
You dont have to use destructuring here, just pass task from this.state directly into edit method